compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -349,6 +350,25 @@ fuseConsumers(RewriterBase &rewriter, Operation *tiledOp,
       continue;
```

Comment:
Can we add a basic check before the tileAndFuse that skips non-linalg ops with more than one use of `loops[0]`

Reply:
Not sure I understand why ?

Reply:
Because IIUC the fusion happens regardless of whether it is a linalg op or not, but we don't do the fixup for non-linalg ops.